src/vec.rs

```
@@ -485,4 +612,228 @@ mod tests {
         }
         assert_eq!(wma.in_use(), 0);
     }
+
+    #[test]
+    fn try_with_capacity_in() {
```

It looks like most of the tests stay within the memory limit. I'd suggest adding an over-limit case to make sure the TryReserveError isn’t silently ignored in the implementation.

---

Deleted this test, augmented the test for `with_capacity_in` which already tested that branch.